discordbot/fetch_dives.py

```diff
@@ -0,0 +1,72 @@
+from ansitable import ANSITable
+
+
+def _clean_line(line):
+	return [
+		# Return every token
+		token.strip()
+		# delimited by `|`
+		for token in line.split("|")
+		# if it is not empty
+		if token
+	]
+
+def _format_warning(w):
+	# No warning, no special highlighting
+	if w == "None":
+		return "{0}".format(w)
+	# No warning known yet, highlight in white
+	elif w == "TBA":
+		return "({0})".format(w)
+	# Warning found, highlight in red
+	else:
+		return "[{0}]".format(w)
+
+
+def fetch_dives(selftext):

```

Comment:
Keyword `self` is used in classes. This `selftext` naming is confusing as it implies this function is supposed to be a method of a class instead and somehow it got to be a function, perhaps by mistake. An alternative name could be `text`, even though it is very general.